applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -1227,7 +1230,15 @@ create_dialplan_move_ccvs(Node, UUID, DP) ->
 -spec create_dialplan_move_ccvs(wh_proplist(), wh_proplist()) -> wh_proplist().
 create_dialplan_move_ccvs(DP, Props) ->
     lists:foldr(
-      fun({<<"variable_", ?CHANNEL_VAR_PREFIX, Key/binary>>, Val}, Acc) ->
+      fun({<<"variable_", ?CHANNEL_VAR_PREFIX, ?CHANNEL_VARS_EXT, _Key/binary>>, _Val}, Acc) ->
```

Comment:
Would this be better in a named function now? It doesn't seem like we're importing variables from the outer scope so it should be fine.